dev/release-notebooks/en/3.5-release-notebook.md

```diff
@@ -159,3 +159,17 @@ The following major changes have been made to the REST API:
 - Plugins wishing to modify the handling of requests to existing API endpoints should follow [a new hook-based pattern](https://github.com/pkp/pkp-lib/issues/9434).
 
 See all the changes in [3.5 API Reference](/dev/api/ojs/3.5.html).
+
+## Task Scheduler Changes
+
+The following major changes have been made to the Task Scheduler:
+
+- The XML based task schedule registration process has been [removed in favour of Laravel Task Scheduler](https://github.com/pkp/pkp-lib/issues/9678).
+- The old schedule tool `tools/runScheduledTasks.php` has been removed and a new scheduling tool has been introduced as `php lib/pkp/tools/scheduler.php run` .
+- A new section in the config file named `[schedule]` has been introduced to have better control of task scheduler. See [here](https://github.com/pkp/pkp-lib/issues/9678#issue-2110219392) or check the [config.TEMPLATE.inc.php](https://github.com/pkp/ojs/blob/stable-3_5_0/config.TEMPLATE.inc.php) for more details.
+- [Acron](https://github.com/pkp/acron) plugin has been deprecated and removed.
+- Plugins also can not register schedule tasks in XML format anymore but need to use the laravel based scheduler convention. See [here](https://github.com/pkp/pkp-lib/issues/9678#issue-2110219392) for more details and the [pln](https://github.com/pkp/pln) plugin as reference of implementation.
```

Comment:
```suggestion
- Plugins that wish to register scheduled tasks should use Laravel scheduler conventions instead of the XML-based format that was previously supported. See [#9678](https://github.com/pkp/pkp-lib/issues/9678#issue-2110219392) for details, and the [PKP Preservation Network](https://github.com/pkp/pln) plugin as a reference.
```